config/samples/simplekafkacluster_with_k8s_provided_nodeport.yaml

```diff
@@ -0,0 +1,257 @@
+apiVersion: kafka.banzaicloud.io/v1beta1
+kind: KafkaCluster
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: kafka
+spec:
+  headlessServiceEnabled: true
+  zkAddresses:
+    - "zookeeper-client.zookeeper:2181"
+  propagateLabels: false
+  oneBrokerPerNode: false
+  clusterImage: "ghcr.io/banzaicloud/kafka:2.13-2.7.0-bzc.1"
```

Comment:
```suggestion
  clusterImage: "ghcr.io/banzaicloud/kafka:2.13-2.7.0-bzc.2"
```

Reply:
I will do a separate PR for that where I will update all our samples with the new clusterImage version